### PR TITLE
Use a consistent response syntax for 2 of 4 of the endpoints WA-160

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Review by salonishah11 — CHANGES_REQUESTED** [review]
Can you also please update the Martha v3 section in README  with the new response schema for `martha_v3` and the support for DRS v1.0 only going forward?

File: README.md [hunk 1]
```diff
@@ -27,6 +27,70 @@ Production: https://us-central1-broad-dsde-prod.cloudfunctions.net/martha_v2
...
+ name:           string,
+ gsUri:          string,
+ googleServiceAccount: string, [null unless the DOS url belongs to a Bond supported host]
+ signedUrl:      string [always the empty string '']
+```
+
+**NOTE:**
+
+`martha_v3` only handles URLs for servers that support requests via the DRS v1.0 path-prefix
+`https://[hostname]/ga4gh/drs/v1/object/[path]`, and then return DRS v1.0 compatible json metadata.
```
Comment (rsasch) [inline README.md]: nit: JSON is in all-caps in the text above, please make consistent

File: README.md [hunk 2]
```diff
@@ -27,6 +27,70 @@ Production: https://us-central1-broad-dsde-prod.cloudfunctions.net/martha_v2
...
+
+There was an [early substitution recommendation to
+users](https://app.zenhub.com/workspaces/orange-5d680d7e3eeb5f1bbdf5668f/issues/databiosphere/azul/1115), instructing
+them to convert their URL schemes from "dos" to "drs". Some of the underlying servers hosting the DOS/DRS metadata have
+not yet been upgraded to support the DRS request path-prefix and DRS response json metadata, so `martha_v2` still
```
Comment (rsasch) [inline README.md]: see nit for line 62

**Review by rsasch — CHANGES_REQUESTED** [review]
You added documentation for the martha_v3 endpoint, but it also looks like you changed the structure of the response for the fileSummaryV1 endpoint, too?  If so, please update the docs for that, as well.

File: README.md [hunk 3]
```diff
@@ -48,6 +112,7 @@ It will always return an object with the same properties:
  bucket:         string,
  name:           string,
  gsUri:          string,
+ googleServiceAccount: string, [always null]
```
Comment (rsasch) [inline README.md:115]: my bad, you added it 👍

**Review by salonishah11 — APPROVED** [review]
LGTM 👍

**Review by rsasch — APPROVED** [review]

**Pull request merged**